packages/perseus/src/widgets/graded-group/graded-group.tsx

```diff
@@ -386,18 +410,45 @@ export class GradedGroup
                             >
                                 {this.context.strings.hideExplanation}
                             </button>
-                            {/**
-                             * We're passing a couple of props to Renderer that it doesn't
-                             * require as part of {...this.props.hint}.
-                             */}
-                            <Renderer
-                                {...this.props.hint}
-                                ref={this.hintRendererRef}
-                                apiOptions={apiOptions}
-                                linterContext={this.props.linterContext}
-                                strings={this.context.strings}
-                                showSolutions={showSolutions}
-                            />
+
+                            <UserInputManager
```

Comment:
This is for hints...which I think can be interactive? But we don't want that input to be in the global store?